Dockerfile

```diff
@@ -25,13 +25,19 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
     mv tini /usr/local/bin/tini && \
     chmod +x /usr/local/bin/tini
 
+# Install Mysql regex UDFs
+RUN wget --quiet https://github.com/mysqludf/lib_mysqludf_preg/archive/testing.zip && \
+    mkdir lib_mysqludf_preg && \
+    unzip testing.zip -d lib_mysqludf_preg && \
+    cd lib_mysqludf_preg && \
+    ./configure && make install && make installdb && make test;
```

Comment:
Hmm, I think this would need to be installed in the MySQL container.

Adding this will probably require us to build our own MySQL container. Or we could use [this](https://github.com/uqlibrary/docker-mysql-udf-preg)